portfolio/src/main/webapp/styles.css

```diff
@@ -0,0 +1,1167 @@
+*{
```

Comment:
Tip: when I have a long css file, I like to use comments to break it up into sections

E.g.
```
/*
=============================
===== PROJECTS SECTION ======
=============================
*/
```

Reply:
Yeah, 1.2k lines really is a lot. I added those sections in because I do think they help with organization